config/application.rb

```diff
@@ -153,7 +153,16 @@ class Application < Rails::Application
 
     # enables in memory cache store with ttl
```

Comment:
Nit: IMHO we should either remove this comment or support it. Probably the former :)